scripts/aws/cloudformation/tasking-manager.template.js

```diff
@@ -20,7 +20,8 @@ const Parameters = {
   },
   DatabaseDump: {
     Type: 'String',
-    Description: 'Path to database dump on S3'
+    Description: 'Path to database dump on S3',
+    Default: 's3://example-bucket/tasking-manager.sql'
```

Comment:
If we add this then every cloudformation update will attempt to restore a backup from "example-bucket/tasking-manager.sql". Better to leave it blank, and instead add the examples into the descriptions.

Reply:
Makes sense. Let me change it.